src/ontology/mondo-edit.obo

```diff
@@ -448254,27 +448259,17 @@ intersection_of: disease_has_basis_in_dysfunction_of GO:0005652 ! nuclear lamina
 
 [Term]
 id: MONDO:0021107
-name: narcolepsy
-def: "A sleep disorder characterized by a tendency for excessive sleepiness during the day which occurs even after adequate sleep in the nighttime. The persons who suffer from this condition experience fatigue and may fall asleep at inappropriate times during the day." [NCIT:C84489]
+name: hereditary narcolepsy
+def: "An instance of narcolepsy that is caused by an inherited genomic modification in an individual." [MONDO:patterns/hereditary]
```

Comment:
I think this is not correct to reuse this ID because it makes the term more specific. 
This ID should be kept for general narcolepsy, and a new term should be created for "hereditary narcolepsy"

Reply:
this term already had the axiom of 'has characteristic' some inherited on it though, so it was logically defined as hereditary narcolepsy.

Do you still think that this should become the more general class?

Reply:
I think this should become the general class because the definition is for the general class. 
If someone thought they were using this term to mean "hereditary narcolepsy", their annotation would still be correct, though less precise. 
However, if someone used this term to mean the general class, and we change it to "hereditary narcolepsy", their annotation might be incorrect.